contracts/FleekERC721.sol

```diff
@@ -18,6 +21,27 @@ contract FleekERC721 is ERC721, FleekAccessControl {
     event NewTokenExternalURL(uint256 indexed token, string indexed externalURL, address indexed triggeredBy);
     event NewTokenENS(uint256 indexed token, string indexed ENS, address indexed triggeredBy);
 
+    event NewAccessPoint(string indexed apName, uint256 indexed tokenId, address indexed owner);
+    event RemoveAccessPoint(string indexed apName, uint256 indexed tokenId, address indexed owner);
+    event ChangeAccessPointScore(
```

Comment:
Should we move this to another score branch and bring it in then? it's not a big deal just if we want to be minimal or not.

Reply:
As it is a base implementation and can be reworked in the future, would it be worth it to remove from this PR and create another one just carrying the code?